test/unit/blots/inline.js

```diff
@@ -26,4 +27,68 @@ describe('Inline', function () {
       '<p><em>0</em><strong><em>12</em></strong><em>3</em></p>',
     );
   });
+
+  fdescribe('Strikethrough', function () {
+    it('renders attributes when strikethrough tag the first', function () {
```

Comment:
And in another places
```suggestion
    it('renders attributes when strikethrough tag is first', function () {
```

Reply:
Fixed